DataViz.sln

```diff
@@ -9,7 +9,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ChartSamples", "sample\Char
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Tests", "src\Tests\Tests.csproj", "{E8E8B63C-4D8D-4986-A8F1-B7B84326321C}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DesignerTest", "DesignerTest\DesignerTest.csproj", "{E30E4A24-0CC6-4746-80CF-9450B2D936A8}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DesignerTest", "DesignerTest\DesignerTest.csproj", "{E30E4A24-0CC6-4746-80CF-9450B2D936A8}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{B4284A32-4712-4367-8F15-07128A123366}"
+	ProjectSection(SolutionItems) = preProject
+		.editorconfig = .editorconfig
+	EndProjectSection
 EndProject
```

Comment:
Pls rebase this PR on top of current Dev. We already have merged this change in https://github.com/kirsan31/winforms-datavisualization/pull/12

Reply:
Done